hub.go

```diff
@@ -15,6 +18,7 @@ func newHub() *hub {
 	return &hub{
 		queue:    make(queue, 16),
 		channels: make(channels),
+		ticker:   multitick.NewTicker(pingPeriod, time.Millisecond*-1),
```

Comment:
use negative millisecond count here, so all timers are on the same interval (>=0 will set an interval for every subscribe call)